README.md

```diff
@@ -122,9 +122,9 @@ Postgres, Redis and Blobstorage resources are tagged with the following key valu
 
 ```bash
 integreatly.org/clusterID: #clusterid
-integreatly.org/product-name: #rhmi component product name
+integreatly.org/product-name: #rhoam component product name
 integreatly.org/resource-type: #managed/workshop 
-integreatly.org/resource-name: #postgres/redis/blobsorage
+integreatly.org/resource-name: #postgres/redis/blobstorage
 ```
```

Comment:
@cecobask , Hi Tsvetoslav, I'd like to ask about Lines 251-261 in this file (sorry don't know how to write it there). Regarding Subscription. RHMI is still there, but it's related to Integrately. Is it planned to do in different Task, or maybe need one more PR,  to change Subscription name and Package name in Integrately?

Reply:
@valerymo I had the same question and it was answered in this thread: https://chat.google.com/room/AAAAssKWhTI/MLanIkWYHps
It's a breaking change and it will not be in scope for this ticket.